### PR TITLE
Update team.html

### PR DESCRIPTION
**Pull request by cwhanse**
Add partners and logo links.

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -26,6 +29,11 @@ <h1>Team</h1>
...
-    <img alt="DOE SETO logo" class="teamlogo" src="/images/logos/eere_logo_horiz_green_small.png">
+    <img alt="PV Pros logo" class="teamlogo" src="/images/logos/PVPros-logo-340x93-1.png">
+  </div>
+  <div class="teamlogo-wrapper">
+    <img alt="DOE SETO logo" class="teamlogo" src="/images/logos/bay4-logo.png">
   </div>
-</div
+  <div class="teamlogo-wrapper">
+    <img alt="DOE SETO logo" class="teamlogo" src="/images/logos/eere_logo_horiz_green_small.png">
+  </div></div
```
Comment (wholmgren) [inline team.html]: looks like this last `</div>` tag is missing the closing `>` in both the original file and this PR.

File: team.html [hunk 2]
```diff
@@ -26,6 +29,11 @@ <h1>Team</h1>
     <img alt="University of Arizona logo" class="teamlogo" src="/images/logos/ua_horiz_rgb_small.png">
   </div>
   <div class="teamlogo-wrapper">
-    <img alt="DOE SETO logo" class="teamlogo" src="/images/logos/eere_logo_horiz_green_small.png">
+    <img alt="PV Pros logo" class="teamlogo" src="/images/logos/PVPros-logo-340x93-1.png">
+  </div>
+  <div class="teamlogo-wrapper">
+    <img alt="DOE SETO logo" class="teamlogo" src="/images/logos/bay4-logo.png">
```
Comment (wholmgren) [inline team.html]:
```suggestion
    <img alt="Bay4 logo" class="teamlogo" src="/images/logos/bay4-logo.png">
```

**Review by wholmgren — COMMENTED** [review]
looks like you'll need to run `git add` on the new logo files

**Comment by wholmgren** [comment]
whoops. just saw you had it covered in https://github.com/SolarPerformanceInsight/SolarPerformanceInsight.github.io/pull/6

**Comment by cwhanse** [comment]
thanks @wholmgren

**Pull request merged**